### PR TITLE
chore: remove angularjs integration

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

File: apps/demos/package.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
         "@angular/platform-browser": "12.2.17",
         "@angular/platform-browser-dynamic": "12.2.17",
         "@aspnet/signalr": "1.0.27",
-        "angular": "1.8.3",
         "angular-sanitize": "1.8.3",
```
Comment (dxvladislavvolkov) [inline apps/demos/package.json]: It seems this can also be removed

File: packages/devextreme/testing/systemjs-builder.js [hunk 2]
```diff
@@ -243,15 +242,6 @@ const transpileJsVendors = async() => {
         }),
     );
 
-    await Promise.all(
```
Comment (ivanblinov2k17) [inline packages/devextreme/testing/systemjs-builder.js:246]: Do you have any ideas, why we even need this file in DevExtreme? I've found it in additional qunit build test task, but the purpose for me is unclear for now

**Review by ivanblinov2k17 — DISMISSED** [review]

File: packages/devextreme/testing/systemjs-builder.js (hunk 2 above)
Comment (VasilyStrelyaev) [reply packages/devextreme/testing/systemjs-builder.js:246]: No, no idea, maybe @alexslavr knows

Comment (alexslavr) [reply packages/devextreme/testing/systemjs-builder.js:246]: Default system.js builder doesn't support SCP. For more details see https://github.com/DevExpress/DevExtreme/pull/24219

**Review by mpreyskurantov — APPROVED** [review]

**Review by ivanblinov2k17 — APPROVED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**